src/main/java/org/terasology/manualLabor/systems/ShearingSystem.java

```diff
@@ -0,0 +1,111 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.manualLabor.systems;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.terasology.engine.core.Time;
+import org.terasology.engine.entitySystem.entity.EntityManager;
+import org.terasology.engine.entitySystem.entity.EntityRef;
+import org.terasology.engine.entitySystem.event.ReceiveEvent;
+import org.terasology.engine.entitySystem.prefab.Prefab;
+import org.terasology.engine.entitySystem.systems.BaseComponentSystem;
+import org.terasology.engine.entitySystem.systems.RegisterSystem;
+import org.terasology.engine.logic.characters.events.AttackEvent;
+import org.terasology.engine.logic.delay.DelayManager;
+import org.terasology.engine.logic.delay.PeriodicActionTriggeredEvent;
+import org.terasology.engine.registry.In;
+import org.terasology.engine.rendering.assets.material.Material;
+import org.terasology.engine.rendering.assets.skeletalmesh.SkeletalMesh;
+import org.terasology.engine.rendering.logic.SkeletalMeshComponent;
+import org.terasology.gestalt.assets.ResourceUrn;
+import org.terasology.gestalt.assets.management.AssetManager;
+import org.terasology.manualLabor.components.ShearableComponent;
+
+import java.util.Optional;
+
+/**
+ * This system is responsible for handling the logic for shearing. A sheep shearing cycle consists of a shearing switching to the model for
+ * sheared state, dropping an item in event of shearing and switching back to the non sheared state after a certain amount of time.
```

Comment:
> dropping an item in event of shearing

This is not part of this PR yet but will added in a follow-up.
Documentation should match the current state of the code.
For future additions it's good practise to add a `TODO` comment or an issue linking to the respective code part or both.
As you already have a PR open for adding the drop you don't need either here, though, but documentation about an item being dropped should only be added when you really add it anyway.